test/map.spec.js

```diff
@@ -150,119 +100,130 @@ test('fillGridWithSubgrid: throws errors for invalid inputs', t => {
 
 
 test('getTileTraversabilityInCells: returns correctly with entirely traversable tile, CPTL=1', t => {
+  setupTiles(true);
   MapRewireAPI.__Rewire__('CPTL', 1);
   const expected = [
     [1],
   ];
-  t.same(getTileTraversabilityInCells(tileTypes.REGULAR_FLOOR), expected);
-
+  t.same(getTileTraversabilityInCells(getId('REGULAR_FLOOR')), expected);
+  teardownTiles();
   t.end();
 });
 
 
 test('getTileTraversabilityInCells: returns correctly with entirely nontraversable tile, CPTL=1', t => {
+  setupTiles(true);
   MapRewireAPI.__Rewire__('CPTL', 1);
   const expected = [
     [0],
   ];
-  t.same(getTileTraversabilityInCells(tileTypes.SQUARE_WALL), expected);
+  t.same(getTileTraversabilityInCells(getId('SQUARE_WALL')), expected);
+  teardownTiles();
   t.end();
 });
 
 
 test('getTileTraversabilityInCells: returns correctly with entirely traversable tile, CPTL=4', t => {
+  setupTiles(true);
   MapRewireAPI.__Rewire__('CPTL', 4);
   const expected = [
     [1, 1, 1, 1],
     [1, 1, 1, 1],
     [1, 1, 1, 1],
     [1, 1, 1, 1],
   ];
-  t.same(getTileTraversabilityInCells(tileTypes.INACTIVE_PORTAL), expected);
-
+  t.same(getTileTraversabilityInCells(getId('INACTIVE_PORTAL')), expected);
+  teardownTiles();
   t.end();
 });
 
 
 test('getTileTraversabilityInCells: returns correctly with CNTO, CPTL=4', t => {
+  setupTiles(true);
   const expected = [
     [1, 0, 0, 1],
     [0, 0, 0, 0],
     [0, 0, 0, 0],
     [1, 0, 0, 1],
   ];
-  t.same(getTileTraversabilityInCells(tileTypes.SPIKE), expected);
-
+  t.same(getTileTraversabilityInCells(getId('SPIKE')), expected);
+  teardownTiles();
```

Comment:
Yikes, probably also my fault: do you have to call that thing to cleanup a rewire every time?
`MapRewireAPI.__ResetDependency__('<VARIABLE>');`?
If so, you could add those in or make an issue and I could do it later?